core/openapi.yaml

```
@@ -471,6 +471,19 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Allowance"
+        scheduled_points:
+          type: array
+          items:
+            type: object
+            properties:
```

please make position and time mandatory
```suggestion
            required:
              - position
              - time
            properties:
```